css/main.css

```diff
@@ -1,3 +1,663 @@
+/*![Juice Bar homepage](./designs/Juice%20bar.png)
+
+Live demo here: https://www.wix.com/website-template/view/html/2962  
+### Fonts
+**Headings and subheadings:** Questrial - Regular 400  
+https://fonts.google.com/specimen/Questrial  
+**Buttons:** Raleway - Regular 400  
+https://fonts.google.com/specimen/Raleway  
+**Body:** Roboto - Light 300  
+https://fonts.google.com/specimen/Roboto  
+
+
 /* Add your CSS here */
 
-/* Dont' forget to link this file to your HTML in the <head> */
+:root {
+  --black: #000;
+  --white: #fff;
+  --lylac: #f5e8ff;
+  --light-green: #e6fac0;
+  --medium-green: #5e7d1f;
+  --dark-green: #394b2a;
+}
+
+body {
+  font-family: "Roboto" sans-serif;
+  font-size: 15px;
+  margin: 0;
+  line-height: 1.38; 
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  text-align: center; 
+}
+
+h1,
+h2,
+h3 {
+  font-family: "Questrial";
+  font-weight: 300;
+  margin: 0;
+}
+
+h2 {
+  font-size: 32px;
+}
+
+h3 {
+  font-size: 22px;
+}
+
+p {
+  font-family:"HelveticaNeueW01-45Ligh";
+  font-size: 15px;
+  font-weight: 70;
+  margin: 0.5rem 2.2rem;
+  line-height: 1.7;
+  letter-spacing:0.2px;
+}
+
+/* Header */
+
+button {
+  font-family: "Raleway";
+  width: 170px;
+  height: 48px;
+  font-size: 15px;
+  border-radius: 0;
+  background-color: var(--black);
+  color: var(--white);
+}
+
+.header .order-btn {
+  background-color: var(--black);
+  width: 100%;
+  color: var(--white);
+}
+
+.login {
+  display: none;
+}
+
+.header-bottom {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  text-align: left;
+  padding: 1rem 1.3rem;
+}	
+
+.bar {
+  display: block;
+  width: 35px;
+  height: 3.5px;
+  margin: 5px auto;
+  background-color: #101010;
+}
+
+/* main / main-top  */
+
+
+.main-top {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 2rem;
+}
+
+.main-top-item {
+  margin: 0.8rem 1.7rem;
+}
+
+.main-top-img {
+  margin-top: 1rem;
+  width: 320px;
+  height: 200px;
+  object-fit: cover;
+  object-position: 50% 50%;
+}
+
+.main-top-try p {
+  padding: 0;
+}
+
+.main-top-try {
+  background-color: var(--lylac);
+  font-size: 16px;
+  padding: 0;
+  width: 100%;
+}
+
+a:link {
+  color: var(--black);
+}
+
+/*  main / main-middle */
+
+.main-middle {
+  margin-bottom: 2rem;
+  text-align: center;
+}
+
+.icons img {
+  width: 4.5rem;
+  height: auto;
+  margin: 1rem auto;
+}
+
+/* main / main-article */
+
+.main-article {
+  background-color: var(--light-green);
+}
+
+.main-article img {
+  width: 280px;
+  height: 377px;
+  object-fit: cover;
+  object-position: 80% 50%;
+  margin: 1.5rem;
+}
+
+ .article-btn {
```

Comment:
can you think how you could rename this class following the BEM conventions we discussed?

Reply:
Thanks Ellie, I change it with " main-article__btn".